keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -56,6 +66,31 @@ Consider the following scenario:
 
 In this case, the administrator would like the evicted workload to be requeued as soon as possible on the newly available capacity.
 
+#### Story 2
+
+In the story 1 scenario, when we set `waitForPodsReady.requeuingStrategy.timestamp=Creation`, 
+the workload endlessly or repeatedly can be put in front of the queue after eviction in the following eviction reasons:
+
+1. The workload don't have the proper configurations like image pull credential and pvc name, etc.
+2. The cluster can meet flavorQuotas, but each node doesn't have the resources that each podSet requests.  
+3. Multiple flavors are matched for the workload, but the workload can not be launched on the backed flavors (which means non-primary flavor).
```

Comment:
was this always there? I don't really understand it.

Reply:
I added this story built with https://github.com/kubernetes-sigs/kueue/pull/1608/files#r1460075048.

Reply:
oh right. Let me rephrase:

```
If there are multiple resource flavors that match the workload (for example, flavors 1 & 2) and the workload was running on flavor 2, it's likely that the workload will be readmitted on the same flavor.
```

Reply:
Thank you for this suggestion!

Reply:
Done.